apps/docs/src/routes/core/object3d.md

```diff
@@ -15,12 +15,12 @@ You might want to use this component to pass as a reference to other components:
 
 ```svelte
 <script>
-  import { Object3D, PerspectiveCamera } from '@threlte/core'
+  import { T, Object3D } from '@threlte/core'
 
   let target
 </script>
 
-<PerspectiveCamera lookAt={target} />
+<T.PerspectiveCamera lookAt={target} />
 <Object3D bind:object={target} />
 ```
```

Comment:
`lookAt` is not a native property to `<T>` (but can be added via the Plugin API #214 by the user or an "official" Plugin). We'll remove everything related for now.

Reply:
This file is no longer present once the `Object3D` removal is merged, so maybe we can leave it as is?

Reply:
Yep, you're right, it's gone :)